projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Companies.php

```diff
@@ -115,278 +115,116 @@ public function html( $company, $metabox ) {
+					$fields               = $zbsCompanyFields; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+					$show_id              = (int) $zbs->settings->get( 'showid' );
+					$fields_to_hide       = $zbs->settings->get( 'fieldhides' );
+					$show_addresses       = (int) zeroBSCRM_getSetting( 'showaddress' );
+					$show_second_address  = (int) zeroBSCRM_getSetting( 'secondaddress' );
+					$show_country_fields  = zeroBSCRM_getSetting( 'countries' );
+					$second_address_label = zeroBSCRM_getSetting( 'secondaddresslabel' );
                 if ( empty( $second_address_label ) ) {
                   $second_address_label = __( 'Second Address', 'zero-bs-crm' );
                 }
-
-               // PerfTest: zeroBSCRM_performanceTest_finishTimer('custmetabox-dataget');
-               // PerfTest: zeroBSCRM_performanceTest_startTimer('custmetabox-draw');
-            
-                //sticky tape some CSS until new UI!!
             ?>
-                <style>
-                    #post-body-content{
-                        display:none;
-                    }
-                        @media all and (max-width:699px){
-                        table.wh-metatab{
-                            min-width:100% !important;
-                        }
-                    }  
-                </style>
                 <script type="text/javascript">var zbscrmjs_secToken = '<?php echo esc_js( wp_create_nonce( 'zbscrmjs-ajax-nonce' ) ); ?>';</script>
 
                 <?php #} Pass this if it's a new customer (for internal automator) - note added this above with DEFINE for simpler.
 
                     if (gettype($zbsCompany) != "array") echo '<input type="hidden" name="zbscrm_newcompany" value="1" />';
 
                 ?>
-
-
-                <table class="form-table wh-metatab wptbp" id="wptbpMetaBoxMainItem">
+			<div>
+				<div class="jpcrm-form-grid" id="wptbpMetaBoxMainItem">
 
                     <?php #} WH Hacky quick addition for MVP 
                     # ... further hacked
 
-                    if ($zbsShowID == "1" && $companyID > 0) { ?>
-                    <tr class="wh-large"><th><label><?php echo esc_html( $this->coOrgLabel ) . ' '; esc_html_e("ID","zero-bs-crm");?>:</label></th>
-							<td style="font-size: 20px;color: black;vertical-align: top;">
-                        #<?php echo esc_html( $companyID ); ?>
-                    </td></tr>
+				if ( $show_id === 1 && $companyID > 0 ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+					?>
+							<div class="jpcrm-form-group">
+								<label class="jpcrm-form-label" for="maintel"><?php echo esc_html( $this->coOrgLabel ) . ' '; esc_html_e( 'ID', 'zero-bs-crm' ); // phpcs:ignore Generic.Formatting.DisallowMultipleStatements.SameLine, WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase, Squiz.PHP.EmbeddedPhp.MultipleStatements ?>:</label>
```

Comment:
Remove incorrect `for`:
```suggestion
								<label class="jpcrm-form-label"><?php echo esc_html( $this->coOrgLabel ) . ' '; esc_html_e( 'ID', 'zero-bs-crm' ); // phpcs:ignore Generic.Formatting.DisallowMultipleStatements.SameLine, WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase, Squiz.PHP.EmbeddedPhp.MultipleStatements ?>:</label>
```